### ISSUE TITLE
V1.2 : "Segmentation fault (core dumped)" error

### ISSUE THREAD

**Issue by TiffanyNeat** [uhwot/lbp_archive_dl#2]
```
thread 'main' panicked at src/main.rs:167:49:
called `Result::unwrap()` on an `Err` value: InvalidStringLength
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
get this error when running any equivalent of
`./archive_dl ./61c5b4dbfb137163d24fcc4480855a4ef3abca5e ./a`
in the terminal. Tried this version- opening terminal/cd'ing @ the path of the files & folder, as well as when trying from root directory with the full paths to each individually listed. there are no spaces in any of the directories, and i also tried the command with inputting <root level> as a working link to the file in the archive.org origin. everything i try leads to the above error, and I can't figure out what it's mad here about... :/ 

for reference i'm trying with a random level i searched for in LBP1, i've my bigfart extracted correctly and all. [link to the level as a singular search result on the LBP Find website, for reference.](https://maticzpl.xyz/lbpfind/?user=&name=wilmas+whisky)

Thanks if you do healp. :heart:

**Comment by uhwot** [comment]
the rootLevel should be `61c5b4dbfb137163d24fcc4480855a4ef3abca5e`, without the ./ at the start

**Comment by TiffanyNeat** [comment]
> the rootLevel should be `61c5b4dbfb137163d24fcc4480855a4ef3abca5e`, without the ./ at the start

Doing that, I get a new error that's simply
`Segmentation fault (core dumped)`
Like the prior, nothing that I do seems to counteract this one. Just keeps happening, regardless.

**Comment by uhwot** [comment]
this should now be fixed on the 1.3 release :)

**Comment by TiffanyNeat** [comment]
it is!!! thank you vey much
